content/id/docs/concepts/workloads/pods/pod-preset.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Pod Preset
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Halaman ini menyajikan gambaran umum tentang `Pod Preset`, yang merupakan objek untuk memasukkan informasi tertentu ke dalam `Pod` pada saat waktu penciptaan. Informasi dapat berupa _secret_, _volume_, _volume mount_, dan variabel _environment_.
+{{% /capture %}}
+
+{{% capture body %}}
+## Memahami **Pod Preset**
+---
+
+Sebuah `Pod Preset` adalah sebuah _resource_ API untuk memasukkan kebutuhan _runtime_ tambahan ke dalam sebuah _Pod_ pada saat waktu penciptaan. Kamu akan menggunakan _label selector_ untuk menunjuk _Pod-pod_ dimana _Pod Preset_ diterapkan.
+
+Menggunakan sebuah _Pod Preset_ memungkinkan pembuat templat _pod_ untuk tidak menyediakan secara eksplisit semua informasi untuk setiap _pod_. Dengan demikian, pembuat templat _pod_ mengkonsumsi sebuah _service_ spesifik untuk tidak perlu tahu semua detail-detail tentang _service_ tersebut.
+
+Untuk informasi lebih lanjut mengenai latar belakang lihat [proposal desain untuk _PodPreset_](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/service-catalog/pod-preset.md).
+
+## Bagaimana Cara Kerja **Pod Preset**
+---
+
+_Kubernetes_ menyediakan sebuah _admission controller_ (**PodPreset**) dimana, ketika diaktifkan, `Pod Preset` diterapkan kepada permintaan _pod_ yang datang. Ketika sebuah penciptaan _pod_ terjadi, sistem melakukan hal-hal berikut:
```

Comment:
Kubernetes doesn't have to be italic

Reply:
fixed in https://github.com/kubernetes/website/pull/14394/commits/bb4445855c8266473904a9566e5791344b91e11d